src/Screens/Search/components/AutocompleteModal.tsx

```diff
@@ -59,40 +58,30 @@ const AutocompleteModal: FunctionComponent<ModalWFinalFormProps> = ({
     const response = await handleTypeAhead(queryString);
     if (response.data) {
       setSuggestions(response.data);
-      setLoading(false);
```

Comment:
The loading indicator is. meant to depict network activity because the locations are fetched over a network, and can be delayed sometimes.

Reply:
yeah, i think i prefer it without the indicator though. i checked some other apps and they don't have it either, they just update the results